bucket/foobar2000.json

```diff
@@ -18,34 +18,36 @@
     "persist": [
         "configuration",
         "index-data",
-        "library"
+        "library",
+        "user-components"
     ],
     "installer": {
         "script": [
             "Remove-Item \"$dir\\`$*\", \"$dir\\uninstall.exe\" -Force -Recurse",
-            "New-Item \"$dir\\portable_mode_enabled\" -Force | Out-Null",
-            "if(!(Test-Path \"$persist_dir\\playlists\")) {",
-            "    New-Item \"$persist_dir\\playlists\" -ItemType Directory -Force | Out-Null",
+            "New-Item \"$dir\\portable_mode_enabled\" | Out-Null",
+            "if(!(Test-Path \"$persist_dir\")) {",
             "    if (Test-Path \"$env:Appdata\\foobar2000\") {",
             "        Write-Host -F yellow \"Copying old '$env:Appdata\\foobar2000' to '$persist_dir'\"",
-            "        Copy-Item \"$env:Appdata\\foobar2000\\*\" \"$dir\" -Recurse -Force",
+            "        Copy-Item \"$env:Appdata\\foobar2000\\*\" \"$dir\" -Recurse",
```

Comment:
I'm against this kind of data migrations in install script because user doesn't have any control over what is happening.
This should be removed and replaced with notice, like in [vscode-portable](https://github.com/lukesampson/scoop-extras/blob/1ef0112d03f4b55fbd4720cf670df4af5098a7f0/bucket/vscode-portable.json#L10-L12).

Reply:
This script won't remove any data.

Reply:
It is still a bad idea to copy data from folder you have no control. You can easily break installation that way.
I'm actually affected by this, because without warning installation script changed in newer version and it copied **everything** from this folder.
There can be other files beside folder this script trying to persist.

Reply:
Other problem is: this script is akin to **chocolatey** packages, and **scoop** isn't **chocolatey**.